spring-file/jmx-run.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
Why not bash?

Reply:
I don't know exactly the difference between bash and sh, but as I understand bash is more featured sh therefore it would be better to use bash here